portfolio/src/main/webapp/script.js

```diff
@@ -81,8 +81,39 @@ function hasWon(){
     }
 }
 
-async function getHello() {
-  const response = await fetch('/data');
-  const hello = await response.text();
-  document.getElementById('hello-container').innerText = hello;
-}
\ No newline at end of file
+/**
+ * Fetches the current state of the game and builds the UI.
+ */
+async function getCommentSection() {
+  // Clear history of comments. 
+  const history = document.getElementById('history');
+  while (history.firstChild) {
```

Comment:
What you're doing should work, but I think you can also just do `history.innerHTML = ''` to remove all HTML elements under history

Reply:
fixed